### ISSUE TITLE
UI scaling 

### ISSUE THREAD

**Issue by vlamacko** [grocy/grocy-desktop#19]
The UI scaling on my 13inch laptop is too zoomed in out-of-the-box. I cannot find a way to adjust it. Both versions, MS Store and Github release, were tested. [See a screenshot.](https://i.imgur.com/R0uCrOg.png)

**Comment by vlamacko** [comment]
Best workaround I found so far, `data\custom_css.html`:
```
<style>
#page-content {
   zoom: 0.7;
   -moz-transform: scale(0.7);
   -moz-transform-origin: 0 0;
}
</style>
```
Will scale the main content, everything seems to work quite well. It won't touch he side navbar though. Using that on the whole body will scale the navbar but it adjust it's height accordingly.

**Comment by berrnd** [comment]
You probably use a DPI scaling option other than 100% (in Windows settings)?

The embedded browser (CefSharp) automatically takes this into account because of
https://github.com/grocy/grocy-desktop/blob/793fb106d33858ee2b614275056c23a7f15b1c27/grocy-desktop/FrmMain.cs#L50

Let's add options to customize the zoom level (or enable default Ctrl+Mousewheel gestures, if possible in CefSharp).

**Comment by vlamacko** [comment]
@berrnd Yes that is the case (150%).

**Comment by berrnd** [comment]
I added a new menu (top menu bar) for this and handling of the default key shortcuts (`Ctrl` + `+`/`-`/`0`).

(Mousewhell events seem to be a bit tricky with CefSharp, so Ctrl+MouseWheel does not work unfortunately.)

That's included in the v1.8.0 release, already ready here, the Microsoft Store release usually takes a couple of days due to verfication.
